File: ABC090/qc.cpp
#include <iostream>
#include <vector>
#include <queue>
#include <cmath>
#include <algorithm>

#define SORT(v, n) sort(v, v+n);
#define VSORT(v) sort(v.begin(), v.end());
#define INF 999999999
using namespace std;

int main() {
  long long n,m;
  long long num = 0;
  cin >> n >> m;
  if (n > 1 && m > 1) {
    num = (n-2) * (m-2);
  } else if (n == 1 && m > 1){
    num = m -2;
  } else if (m == 1 && n > 1) {
    num = n-2;
  } else if (m == 1 && n == 1) {
    num = 1;
  } else {
  }
  cout << num << endl;
}

File: ABC155/qc.py
def main():
    N = int(input())
    S = [input() for x in range(N)]
    S.sort()
    num = 0
    m = 0
    s = [S[0]]
    for i in range(1, N):
        if S[i-1] == S[i]:
            num = num + 1
        else:
            num = 0
        if num > m:
            m = num
            s = [S[i]]
        elif num == m:
            s.append(S[i])
    # print(s)
    for str in s:
        print(str)


if __name__ == '__main__':
    main()
